docs/configuration.md

```diff
@@ -74,6 +74,7 @@ The following subsections cover all CLI flags which promscale supports. You can
 | db.connections.reader-pool.size               | integer  | 30% of possible connections db | Maximum size of the reader pool of database connections. This defaults to 30% of max_connections allowed by the database.                                                      |
 | db.connections.writer-pool.size               | integer  | 50% of possible connections db | Maximum size of the writer pool of database connections. This defaults to 50% of max_connections allowed by the database.                                                      |
 | db.connections.writer-pool.synchronous-commit | boolean  |             false              | Enable/disable synchronous_commit on database connections in the writer pool.                                                                                                  |
+| db.connections.maint-pool.size                | integer  |               5                | Maximum size of the maintenance pool of database connections. This defaults to 5                                                                                               |
```

Comment:
Should we specify that `vacuum.parallelism` takes the connections from this pool? Or some guidelines like it should be at least `vacuum.parallelism` + whatever else

Reply:
👍